test/map-symbolic/lookup-3-spec.k.out.golden

```diff
@@ -18,7 +18,7 @@
     true
   #Equals
     Z:MyId in_keys ( MAP
-    ( Y:MyId |-> 2 ) )
+    ( Y:MyId |-> 1 ) )
```

Comment:
This doesn't look right, we have `X ==K Y`, and we update the value `Y` points to, so it should be `2` not `1`, right?

Reply:
The meaning of the output is unchanged. The Map appears under `in_keys`, so actually the values are irrelevant.